cypress/e2e/hermioneAccount.spec.cy.js

```
@@ -0,0 +1,87 @@
+/// <reference types='cypress' />
+
+import { faker } from '@faker-js/faker';
+
+describe('Bank app', () => {
+  const user = 'Hermoine Granger';
+  const accountNumber = '1001';
+  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
+  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
+  let initialBalance;
+
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.login(user);
+
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', '5096')
```

Using a hardcoded value like `'5096'` in your selector makes the test brittle. If the initial balance changes for this account, this selector will fail. Since you are already dynamically capturing the balance in the next step, consider using a selector that doesn't depend on a specific value to make your test more robust.